src/pipeline/rackhd/docker/Build.groovy

```diff
@@ -0,0 +1,51 @@
+package pipeline.rackhd.docker
+
+def build(String manifest_path, String clone_dir, String workdir){
+    def ret_dict=[:]
+    node(build_docker_node){
+        withEnv([
+            "MANIFEST_FILE_URL=${env.MANIFEST_FILE_URL}",
+            "IS_OFFICIAL_RELEASE=${env.IS_OFFICIAL_RELEASE}",
+            "ARTIFACTORY_URL=${env.ARTIFACTORY_URL}",
+            "STAGE_REPO_NAME=${env.STAGE_REPO_NAME}"]){
+            deleteDir()
+            def current_workspace = pwd()
+            String library_dir = "./on-build-config"
+            def shareMethod = new pipeline.common.ShareMethod()
+            shareMethod.checkoutOnBuildConfig(library_dir)
+            withCredentials([
+                usernamePassword(credentialsId: 'ff7ab8d2-e678-41ef-a46b-dd0e780030e1',
+                                 passwordVariable: 'SUDO_PASSWORD',
+                                 usernameVariable: 'SUDO_USER')]){
+                    timeout(90){
+                        withEnv(["WORKSPACE=${current_workspace}"]){
+                            sh """#!/bin/bash -ex
+                            bash $library_dir/src/pipeline/rackhd/docker/build.sh \
+                                   --MANIFEST_FILE_URL  ${manifest_path} \
+                                   --ARTIFACTORY_URL ${ARTIFACTORY_URL} \
+                                   --STAGE_REPO_NAME ${STAGE_REPO_NAME} \
+                                   --CLONE_DIR ${clone_dir} \
+                                   --IS_OFFICIAL_RELEASE ${IS_OFFICIAL_RELEASE} \
+                                   --WORKDIR ${workdir}
+                            """
+                        }
+                    }
+                    archiveArtifacts 'rackhd_docker_images.tar, build_record'
+                    stash name: 'docker', includes: 'rackhd_docker_images.tar, build_record'
+                    env.DOCKER_WORKSPACE="${current_workspace}"
+                    env.DOCKER_STASH_NAME="docker"
+                    env.DOCKER_STASH_PATH="rackhd_docker_images.tar"
+                    env.DOCKER_RECORD_STASH_PATH="build_record"
```

Comment:
The information in file "build_record" can be read from the ouput of the command:
```docker load -i *.tar```
So, I'm thinking if we can remove the file...